lib/teiserver/battle/balance/split_one_chevs.ex

```diff
@@ -1,53 +1,99 @@
 defmodule Teiserver.Battle.Balance.SplitOneChevs do
   @moduledoc """
-    This balance algorithm first sorts the users by visible OS (match rating) descending. Then all rank=0 (one chevs) will be placed at the bottom of this sorted list.
+  Overview:
+  The goal of this algorithm is to mimic how a human would draft players given the visual information in a lobby.
+  Humans will generally avoid drafting overrated new players.
 
-    Next a team will be chosen to be the picking team. The picking team is the team with the least amount of players. If tied, then the team with the lowest total rating.
+  Details:
+  The team with the least amount of players will pick an unchosen player. If there are multiple teams tied for
+  the lowest player count, then the team with the lowest match rating picks.
 
-    Next the picking team will pick the player at the top of the sorted list.
+  Your team will prefer 3Chev+ players with high OS. If your team must pick a 1-2Chev player,
+  it will prefer lower uncertainty.
 
-    This is repeated until all players are chosen.
+  This is repeated until all players are chosen.
 
-    This algorithm completely ignores parties.
+  This algorithm completely ignores parties.
 
   """
   alias Teiserver.Battle.Balance.SplitOneChevsTypes, as: ST
   alias Teiserver.Battle.Balance.BalanceTypes, as: BT
+  import Teiserver.Helper.NumberHelper, only: [format: 1]
+
+  @splitter "---------------------------"
 
   @doc """
   Main entry point used by balance_lib
   See split_one_chevs_internal_test.exs for sample input
   """
   @spec perform([BT.expanded_group()], non_neg_integer(), list()) :: any()
-  def perform(expanded_group, team_count, _opts \\ []) do
-    members = flatten_members(expanded_group) |> sort_members()
-    %{teams: teams, logs: logs} = assign_teams(members, team_count)
-    standardise_result(teams, logs)
+  def perform(expanded_group, team_count, opts \\ []) do
+    if has_enough_noobs?(expanded_group) do
+      members = flatten_members(expanded_group) |> sort_members()
+      %{teams: teams, logs: logs} = assign_teams(members, team_count)
+      standardise_result(teams, logs)
+    else
+      # Not enough noobs; so call another balancer
+      result = Teiserver.Battle.Balance.LoserPicks.perform(expanded_group, team_count, opts)
+
+      new_logs =
+        ["Not enough noobs; calling another balancer.", @splitter, result.logs]
+        |> List.flatten()
+
+      Map.put(result, :logs, new_logs)
+    end
+  end
+
+  @spec has_enough_noobs?([BT.expanded_group()]) :: bool()
+  def has_enough_noobs?(expanded_group) do
```

Comment:
`has_enough_noobs` only checks if there are noobs, not how many?
Is this intended or not?

Reply:
I had some ideas like enough noobs be something like 
- At least a single 1chev
- At least two, 2chevs
But decided to keep it simple for now.